### PR TITLE
overview slide description update, and cosmetic fixes.

### PR DESCRIPTION
**Pull request by fujitatomoya**
suggested changes for overview and readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # ros2_load_balancing_service (In development)
+
 This package helps ROS2 services implement load balancing functionality.
 
+see [overview slide deck](https://raw.githack.com/Barry-Xu-2018/ros2_load_balancing_service/main/doc/overview.html) for more information.
```
Comment (fujitatomoya) [inline README.md:5]: this will be visible once this PR is merged into the `main.`

File: doc/overview.md [hunk 2]
```diff
@@ -11,23 +14,17 @@ section {
...
-section {
-    font-size: 25px;
-}
-</style>
-
 # Objective
 
-- Support multiple service servers on the same service path to implement redundancy and load-balancing.
-- Existing ROS2 service server/client programs can be used without code modification.
+- ROS 2 service load-balancing in application layer without protocol change.
```
Comment (fujitatomoya) [inline doc/overview.md:25]: there needs to be a space between `ROS` and `2` as official documentation. `ROS 2`, not `ROS2`...

**Review by fujitatomoya — COMMENTED** [review]
@Barry-Xu-2018 here are suggested fixes for documentation and slide deck. i do not have strong opinion here so please take as much as you like.

File: doc/overview.md [hunk 3]
```diff
@@ -93,38 +72,41 @@ $ ros2 run load_balancing_service load_balancing_service -s add_two_ints -t exam
   Interval to discovery server: 1s
 ------------------------------
 Service client remap service name to /load_balancing/add_two_ints
-Service server remap service name to /load_balancing/add_two_ints/XXX
+Service server remap service name to /load_balancing/add_two_ints/s1
```
Comment (Barry-Xu-2018) [inline doc/overview.md]: This information is output when starting the load balancing server application, as there are no backend service servers connected at this time. Therefore, it always outputs "XXX" here.

**Comment by Barry-Xu-2018** [comment]
@fujitatomoya 

Thank you for your modification, I have no objections. Just one comment, please take a look.

File: doc/overview.md (hunk 3 above)
Comment (fujitatomoya) [reply doc/overview.md]: https://github.com/Barry-Xu-2018/ros2_load_balancing_service/pull/3/commits/3664ddae71bc38f3d2a93b688f432fa752e4b7dc fixes this.

**Pull request merged**